app/views/works/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<main>
+
+  <h2>List of Works</h2>
+
+  <% hash = { "album" => "Albums", "book" => "Books", "movie" => "Movies"} %>
+  <section class="media-table">
+    <% ["album", "book", "movie"].each do |category| %>
+      <% works = @works.where(category:category) %>
+      <% works = works.sort_by { |work| work.votes.count }.reverse! %>
+      <table class="table">
+        <h4><%= hash[category] %></h4>
+        <thead>
+        <tr>
+          <th width="80">Votes</th>
+          <th>Title</th>
+          <th width=28%>Created By</th>
+          <th width="100">Published</th>
+          <th width="100">Upvote</th>
+        </tr>
+        </thead>
+        <tbody>
+        <% works.each do |work| %>
```

Comment:
Again, great use of a loop to DRY up your code. Consider how you could also do this with a partial view.